.readthedocs.yaml

```diff
@@ -0,0 +1,22 @@
+---
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python in the build environment.
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.10"
+
+mkdocs:
+  configuration: "mkdocs.yml"
+  fail_on_warning: false
```

Comment:
Any reason for setting this to false?

Reply:
It's old docs, they have warnings (and fixing requires moving some files around which might break other stuff, so not touching). Figured it's better to have them built as they are rather than not.